docassemble/ALWeaver/data/questions/assembly_line.yml

```diff
@@ -644,6 +699,17 @@ code: |
 
   add_metadata = True
 ---
```

Comment:
Maybe remove `default` from this block? Because if the author later overrides values in this block, then they would not be default anymore, right? 

In other words, we provide some default values for vars, but the vars themselves are not "defaults" to be replaced by other vars.